README.md

```diff
@@ -134,9 +134,9 @@ Check the [`./features.md`](./features.md) for more details.
 
 - `-m`, `--model`: Specify Large Language Model to use.
 - `-f`, `--format`: Ask the LLM to format the response in a given format.
-- `--format-as`: Specify the format for the output (used with `--format`).
-- `-P`, `--prompt`: Prompt should include stdin and args.
-- `-p`, `--prompt-args`: Prompt should only include args.
+- `--format-as`: Specify the format for the output (used with `--format`, e.g. `mods -f --format-as json 'your prompt here'`).
```

Comment:
the other flags don't have example usages in their help, i think this should be somewhere else
```suggestion
- `--format-as`: Specify the format for the output (used with `--format`).
```